modules/distribution-zones/src/test/java/org/apache/ignite/internal/distributionzones/BaseDistributionZoneManagerTest.java

```diff
@@ -124,6 +124,7 @@ void setUp() {
         zonesConfiguration = registry.getConfiguration(DistributionZonesConfiguration.KEY);
 
         distributionZoneManager = new DistributionZoneManager(
+                null,
```

Comment:
It seems weird to me that we have `DistributionZoneCausalityDataNodesTest`, and `DistributionZoneManager#waitZoneProcessing` is never tested - neither there, nor in other places. And this injection of `null` assumes that this method will not work in derived classes. Is it correct that there are no tests for `DistributionZoneManager#waitZoneProcessing`?

Reply:
Fixed. I added a registry instead of null.  `waitZoneProcessing` was removed. Versioned value is used in dataNodes method now, so it is used in many tests.